nrfs/include/services/nrfs_swext.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef NRFS_SWEXT_H
+#define NRFS_SWEXT_H
+
+#include <internal/services/nrfs_swext.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief SWEXT control service response notification types. */
+typedef enum __NRFS_PACKED {
+	NRFS_SWEXT_EVT_ENABLED,		/** Request response. */
```

Comment:
Please start assigning numbers for the services, that will make it easier to retain compatibility if there is a change in the future